var/spack/repos/builtin/packages/libgpg-error/package.py

```diff
@@ -19,3 +19,6 @@ class LibgpgError(AutotoolsPackage):
     version('1.27', sha256='4f93aac6fecb7da2b92871bb9ee33032be6a87b174f54abf8ddf0911a22d29d2')
     version('1.21', sha256='b7dbdb3cad63a740e9f0c632a1da32d4afdb694ec86c8625c98ea0691713b84d')
     version('1.18', sha256='9ff1d6e61d4cef7c1d0607ceef6d40dc33f3da7a3094170c3718c00153d80810')
+
+    depends_on('awk', type=('build'))
+    patch('awk-5.patch', when='@1.36^gawk@5:')
```

Comment:
If we always use this patch, do we still need `depends_on('awk')`?

Reply:
This patch does not eliminate the need for awk, it just considers changes to the gawk interface. Without the dependency spack would use the globally installed awk implementation